tox.ini

```diff
@@ -6,3 +6,6 @@ deps=-rrequirements-test.txt
 commands=
     pytest --cov {envsitepackagesdir}/datauri {posargs} tests/
     flake8 datauri/
+
+[flake8]
+max-line-length = 90
```

Comment:
this seems unrelated but ok

Reply:
Yes, not related to the issue, but with 79 chars I can't even put the link on the source of `CachedProperty` without splitting it. That's silly.